
<html> <head>
<br>
<br>
<p>
<p>



<title>Mike Duff</title>
</head>

<body bgcolor="#445544" text=#99AA99 link=#DD7711 vlink=#DD7711 alink=#DD7711>
<font face='helvetica'>


<TABLE>

<TR><TD width="10%"><TD width=600>
<!-- <img src = "thinking_about_computational_genomics_in_leucadia.jpg" width="300" height="200">

<img src = "duff_labphoto1.jpg" width="300" height="200">
-->
<img src = "ranger_lake.png" width="318" height="342">

<h3>Michael Duff</h3>

<a href="http://genetics.uchc.edu">
<br>
Department of Genetics and Genome Sciences</a><br>
UConn Health Center<br>
Institute for Systems Genomics<br>
University of Connecticut<br>
400 Farmington Ave<br>
Farmington CT 06030<br>
<a href="mailto:moduff@gmail.com"><font color="#DD7711">moduff@gmail.com</font></a>
 
 <p>
 <font color="#FF0000">
  ***I've left UConn for the west coast***
 </font>
 <p><em>
 I'm currently seeking employment as a research scientist in the areas of: Bioinformatics, 
 Computational Genomics, Machine Learning, Bayesian methods, Applied Statistics/Mathematics, 
 Reinforcement Learning Theory, Deep-Learning, Optimization and Operations Research.</em>

<html> <head>
<br>
<br>
<p>
<p>


<a href="http://graveleylab.cam.uchc.edu/Graveley/index.html"> Graveley Lab</a><br><p>
Efforts in the lab have been directed toward bringing computational approaches, statistical modeling methods, and
 bioinformatics tools to bear upon problems posed by the analysis of Next-Generation-Sequencing (NGS) data,
 including the standard repertoire of techniques drawn from computational genomics: genome assembly, sequence alignment, 
 gene-expression analysis, motif finding, SNP and structural-variant detection, phylogenetic analysis, 
 and identification of regulatory sequences and networks---as well as the development of special-purpose 
 ad-hoc computational tools for particular applications and novel techniques for data visualization.
<p>


<ul>

<li>NGS data analysis<p> 

<ul>

 <li>DNA/RNA/ChIP/CLIP/iCLIP/RIP-seq data analysis(<a href="http://genome.ucsc.edu/ENCODE/">ENCODE</a>)<br><p> 

 <li>alternative splicing and recursive splicing in D.melanogaster (<a href="http://www.modencode.org/">modENCODE</a>, DSCAM)<br><p>

 <li>RNA interactions and functional intronic domains<br><p>

 <li>lincRNA data analysis in hES differentiation<br><p>

 <li>Human genetic variation, GWAS, and psychogenetics<br><p>

 <li>analysis of RNAi off-target effects<br><p>

 <li>transplicing<br><p>

 <li>Polyomavirus editing, transcript expression profiling for the virus and its host<br><p>

<!--
 <li>transcription/splicing kinetic models<br><p>
-->
 <li>alternative-splicing changes during Human embryonic stem cell differentiation<br><p>

 <li>Archaeal CRISPR-Cas systems<br><p>
<!--
 <li>Squid (<i>Loligo pealei</i>) genome assembly and RNA-editing analysis<br><p>

 <li>Drosophila neurogenetics<br><p>
-->
</ul>

<li>machine learning methods  <p>

 <ul>
 
  <li>probabilistic graphical models, model-structure induction, deep learning, Bayesian methods<br><p>
  <li>causal inference and experimental design<br><p>
  <li>reinforcement learning theory<br><p>
  <li>sequential decision-making under uncertainty<br><p>
 </ul>
   
   <li>development and teaching of two new courses: 
"Bioinformatics Practicum" and "Machine Learning for Genomics."

<!--

<li>other inquisitions <p>
  <ul> 
    <li>Monte-Carlo computation for gauging independence in very-large contingency tables<br><p>
    <li>dynamic modelling of sepsis<br><p> 
    <li>subgraph isomorphism and spectral graph theory<br><p>
    <li>analytic combinatorics<br><p>
  </ul> 
    
-->

</ul>

<br>
<p>
<br>
<p>
<br>
 
<font color="#FFCC00">
How can Next-Generation-Sequencing be used to infer causal relationships between transcribed elements?  How
 can ambiguity and bias be modeled and resolved?</font>

<br>
<p>
<br>

<h3>Papers</h3>
   
Van Nostrand, E.L., et al (2017) <a href="https://www.biorxiv.org/content/early/2017/08/23/179648">A Large-Scale Binding and Functional Map of Human RNA
Binding Proteins</a> (To appear in Nature). http://www.biorxiv.org/content/early/2017/08/23/179648 
<p>   
Chang, H.H., Eibi, G., et al (2017) <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184455">Incidence of Pancreatic Cancer is Dramatically Increased by a
 High Fat, High Calorie Diet</a> in KrasG12D Mice. PLoS ONE 12(9).
<p>
Pilling, L.C., et al (2017) <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0185083">Red Blood Cell Distribution Width: genetic evidence for aging pathways
in 116,666 volunteers</a>. PLoS ONE 12(9).<p> 
Garren, S., Duff, M.O., & Carmichael, G. (2015) <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1005166">Global analysis of mouse polyomavirus infection reveals dynamic regulation of viral and host gene expression and promiscuous viral RNA editing</a>. PLOS Pathogens, 11(9), September 25.
<p>
Stoiber, M.H. et al  (2015).  <a href="http://genome.cshlp.org/content/early/2015/08/20/gr.182675.114.abstract">Extensive cross-regulation of posttranscriptional regulatory networks in Drosophila</a>, published onlinein Genome Research, Aug 20.
<p>
Brooks, A.N., Duff, M.O., May, G., Yang, L., Landolin, J., Wan, K., Sandler, J., Celniker, S.E., Graveley, B.R. and Brenner, S.E. (2015) <a href="http://genome.cshlp.org/content/early/2015/08/20/gr.192518.115.abstract">Regulation of alternative splicing in Drosophila by 56 RNA binding proteins</a>, published online in Genome Research, Aug 20.
<p>
Duff, M.O., Olson, S, Wei, X,, Osman, A., Plocik, A., Bolisetty, M. Celniker, S. & Graveley, B.R. (2015) <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature14475.html">Genome-wide Identification of Zero Nucleotide Recursive Splicing in Drosophila</a>. Nature 521, May 21, 376-9.
<p>
Gerstein, M.B., et al (2014) <a href="http://www.nature.com/nature/journal/v512/n7515/full/nature13424.html">Comparative analysis of the transcriptome across distant species</a>. Nature 512, 445-448.
<p>
Chen, Z.X. et al (2014) <a href="http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=24985915">Comparative validation of the D. melanogaster modENCODE Transcriptome Annotation</a>. Genome Res. July;24(7):1209-23.
<p>
Brown, J.B. et al (many authors). 2014.<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12962.html"> Diversity and dynamics of the Drosophila transcriptome</a>. Nature, published online March 16, 2014.
<p>
Eipper-Mains JE et al (2013) <a href="http://www.ncbi.nlm.nih.gov/pubmed/23094851">Effects of cocaine and withdrawal on the mouse nucleus accumbens transcriptome</a>. Genes Brain Behav.12:21-33 
<p>
Smibert P, Miura P, Westholm JO, Shenker S, May G, Duff MO, Zhang D, Eads BD, Carlson J, Brown JB, Eisman R
C, Andrews J, Kaufman T, Cherbas P, Celniker SE, Graveley BR, Lai EC. (2012) <a href="http://www.cell.com/c
ell-reports/fulltext/S2211-1247(12)00021-6">Global patterns of tissue-specific alternative polyadenylation in Drosophila</a>. Cell Reports, Feb 23.
<p>
Yang L., Duff MO, Graveley BR, Carmichael GG and Chen L-L. 2011. <a href="http://genomebiology.com/2011/12/
2/R16">Genomewide characterization of long nonpolyadenylated RNAs</a>. Genome Biology, Feb 16. 
<p>
  Roy, S. et al (many authors). 2010. <a href="http://www.sciencemag.org/content/330/6012/1787.full">Identification of functional elements and regulatory circuits in Drosophila by large-scale data integration</a>. 
Science, Dec 24.
<p>
  Graveley, B. et al (many authors). 2010. <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature09715.html">The developmental transcriptome of Drosophila melanogaster</a>. Nature, Dec 22. 
<p>
  Cherbas, L., et al (many authors). 2010. <a href="http://genome.cshlp.org/content/21/2/301.full">The transcriptional diversity of 25 Drosophila cell lines</a>. Genome Research, Dec 22. 
<p>
  Brooks, A.N., Yang. L., Duff, M.O., Hansen, K.D., Dudoit, S., Brenner, S.E., Graveley, B.R. 2010. <a href
="http://genome.cshlp.org/content/early/2010/10/04/gr.108662.110">Conservation of an RNA Regulatory Map between Drosophila and Mammals</a>. Genome Research, Oct 4.  
<p>
  McManus, C.J., Duff, M.O., Eipper-Mains, J., Graveley, B.R. 2010. <a href="http://www.pnas.org/content/107/29/12975.short">Global Analysis of Transplicing in Drosophila</a>. Proc Natl Acad Sci, July 1. 
<p>
  McManus, C.J., Coolon, J.D., Duff, M.O., Eipper-Mains, J., Graveley, B.R. and Wittkopp, P.J. 2010. <a href="http://www.pnas.org/content/107/29/12975.short">Regulatory Divergence in Drosophila Revealed by mRNA-Seq
</a>. Genome Research, 20(6):816-25. 
<p>
  Hale, C.R., Zhao, P., Olsen, S., Duff, M.O., Graveley, B.R., Wells, L., Terns, R.M., and Terns, M.P. 2009
. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19945378">RNA-guided RNA cleavage by a CRISPR RNA-Cas protein complex</a>. Cell, 139(5):945-56. Nov 25. 


<p>
<br>
<p>
<br>
<p>
<br>

<a href="older_papers.html"><font color="#3399FF">Older things</font></a>

<p>
<br>

<a href="chrono_bio.html"><font color="#3399FF">My Erdos number is 3.</font></a>


<br>

</font>

<TD width="20%">
</TABLE>
</body>


  </html>
