<html> <head>

<br>

<br>

<p>

<p>







<title>Mike Duff</title>

</head>



<body bgcolor="#445544" text=#99AA99 link=#DD7711 vlink=#DD7711 alink=#DD7711>

<font face='helvetica'>





<TABLE>



<TR><TD width="10%"><TD width=600>

<!-- <img src = "thinking_about_computational_genomics_in_leucadia.jpg" width="300" height="200">



<img src = "duff_labphoto1.jpg" width="300" height="200">



<img src = "ranger_lake.png" width="318" height="342">

-->

<h3>Michael Duff</h3>

Assistant Professor in Residence<br>

<a href="http://genetics.uchc.edu">

Department of Genetics and Genome Sciences</a><br>

UConn Health Center<br>

Institute for Systems Genomics<br>

University of Connecticut<br>

400 Farmington Ave<br>

Farmington CT 06030<br>

<a href="mailto:moduff@gmail.com"><font color="#DD7711">moduff@gmail.com</font></a>

 

 <p>

 <font color="#FF0000">

  ***As of December 2018, I am with the Fred Hutchinson Cancer Research Center***

  in sunny Seattle WA, conducting research on B-cell receptor repertoire dynamics
  
  in response to infection and vaccination.<br>

	       

 </font>

<p>	
	<a href="moduff.github.io"><font color="#3399FF">moduff.github.io</font></a>   

 <p><em>

Research directions: Machine Learning, 

 Bioinformatics/Computational Biology/Computational Genomics, Reinforcement Learning Theory,

  Bayesian methods, Applied Statistics/Mathematics, 

  AI, Deep-Learning, Optimization and Operations Research.</em>
	 
<!--
  <a href="duff_cv_april_2019.pdf"> CV</a>
-->
	 
<html> <head>

<br>

<br>

<p>

<p>





My efforts in the <a href="http://graveleylab.cam.uchc.edu/Graveley/index.html"> Graveley Lab</a>

have been directed toward bringing computational approaches, statistical modeling methods, and

 bioinformatics tools to bear upon problems posed by the analysis of Next-Generation-Sequencing (NGS) data,

 including the standard repertoire of techniques drawn from computational genomics: genome assembly, sequence alignment, 

 gene-expression analysis, motif finding, SNP and structural-variant detection, phylogenetic analysis, 

 and identification of regulatory sequences and networks---as well as the development of special-purpose 

 ad-hoc computational tools for particular applications and novel techniques for data visualization.

<p>





<ul>



<li>NGS data analysis<p> 



<ul>



 <li>DNA/RNA/ChIP/CLIP/iCLIP/RIP-seq data analysis(<a href="http://genome.ucsc.edu/ENCODE/">ENCODE</a>)<br><p> 



 <li>alternative splicing and recursive splicing in D.melanogaster (<a href="http://www.modencode.org/">modENCODE</a>, DSCAM)<br><p>



 <li>RNA interactions and functional intronic domains<br><p>



 <li>lincRNA data analysis in hES differentiation<br><p>



 <li>Human genetic variation, GWAS, and psychogenetics<br><p>



 <li>analysis of RNAi off-target effects<br><p>



 <li>transplicing<br><p>



 <li>Polyomavirus editing, transcript expression profiling for the virus and its host<br><p>



<!--

 <li>transcription/splicing kinetic models<br><p>

-->

 <li>alternative-splicing changes during Human embryonic stem cell differentiation<br><p>



 <li>Archaeal CRISPR-Cas systems<br><p>

<!--

 <li>Squid (<i>Loligo pealei</i>) genome assembly and RNA-editing analysis<br><p>



 <li>Drosophila neurogenetics<br><p>

-->

</ul>



<li>machine learning methods  <p>



 <ul>

 

  <li>Bayesian methods, model-structure induction, and deep learning, transformers, LLMs, & generative diffusion for (structural) biology<br><p>

  <li>reinforcement learning theory<br><p>

  <li>experimental design<br><p>

  <li>sequential decision-making under uncertainty, bandit problems<br><p>

 </ul>

   

   <li>two new courses: <p>

      <ul>

       <li>MEDS 5325: <em>Bioinformatics Practicum</em> <p>

       <li>MEDS 6498: <em>Machine Learning for Genomics</em> <p>

      </ul>   

       



<!--



<li>other inquisitions <p>

  <ul> 

    <li>Monte-Carlo computation for gauging independence in very-large contingency tables<br><p>

    <li>dynamic modelling of sepsis<br><p> 

    <li>subgraph isomorphism and spectral graph theory<br><p>

    <li>analytic combinatorics<br><p>

  </ul> 

    

-->



</ul>



<br>

<p>

<br>

<p>

<br>

 

<font color="#FFCC00">

How can Next-Generation-Sequencing be used to infer causal relationships between transcribed elements?  How

 can ambiguity and bias be modeled and resolved?</font>



<br>

<p>

<br>

 

 <img src = "computational_genomics_unit.jpg" width="500" height="250">

 <br>

 <p>

 

<h3>Papers</h3>

Willis, J.R. et al (2025). <a href = "https://www.science.org/doi/10.1126/science.adr8382">Vaccination with mRNA-encoded nanoparticles drives early maturation of HIV bnAb precursors in humans, Science 0,eadr8382DOI:10.1126/science.adr8382

<p>

Tom G. Caniels et al. ,Precise targeting of HIV broadly neutralizing antibody precursors in humans.Science 0,eadv5572DOI:10.1126/science.adv5572	
<p></p>
	

<p>
	
Hahn, W.O. et al (2024). <a href = "https://doi.org/10.1101/2024.05.08.24306957">HIV BG505 SOSIP.664 trimer with 3M-052-AF/alum induces human autologous tier-2 neutralizing antibodies</a>. medRxiv 2024.05.08.24306957

<p>
	 
	   	      
Cohen, K.W., et al (2021) <a href="https://www.medrxiv.org/content/10.1101/2021.08.23.21262508v1">Shared HIV envelope-specific B cell clonotypes induced by a pox-protein vaccine regimen</a>. medRxiv doi: https://doi.org/10.1101/2021.08.23.21262508
	      
<p>

Mou, Y.,et al (2020) <a href="https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-01088-0">Impaired lipid metabolism in astrocytes underlies degeneration of cortical projection neurons in hereditary spastic paraplegia</a>, Acta Neuropathologica Communications https://doi.org/10.1186/s40478-020-01088-0

<p>
	
				  
Van Nostrand, E.L., et al (2020) <a href="https://www.nature.com/articles/s41586-020-2077-3">A Large-Scale Binding and Functional Map of Human RNA
	Binding Proteins</a>, Nature 583, pp. 711â€“719, July 29.
				  

<p>   

Chang, H.H., Eibi, G., et al (2017) <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184455">Incidence of Pancreatic Cancer is Dramatically Increased by a

 High Fat, High Calorie Diet in KrasG12D Mice</a>. PLoS ONE 12(9).

<p>

Pilling, L.C., et al (2017) <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0185083">Red Blood Cell Distribution Width: genetic evidence for aging pathways

in 116,666 volunteers</a>. PLoS ONE 12(9).<p> 

Garren, S., Duff, M.O., & Carmichael, G. (2015) <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1005166">Global analysis of mouse polyomavirus infection reveals dynamic regulation of viral and host gene expression and promiscuous viral RNA editing</a>. PLOS Pathogens, 11(9), September 25.

<p>

Stoiber, M.H. et al  (2015).  <a href="http://genome.cshlp.org/content/early/2015/08/20/gr.182675.114.abstract">Extensive cross-regulation of posttranscriptional regulatory networks in Drosophila</a>, published onlinein Genome Research, Aug 20.

<p>

Brooks, A.N., Duff, M.O., May, G., Yang, L., Landolin, J., Wan, K., Sandler, J., Celniker, S.E., Graveley, B.R. and Brenner, S.E. (2015) <a href="http://genome.cshlp.org/content/early/2015/08/20/gr.192518.115.abstract">Regulation of alternative splicing in Drosophila by 56 RNA binding proteins</a>, published online in Genome Research, Aug 20.

<p>

Duff, M.O., Olson, S, Wei, X., Osman, A., Plocik, A., Bolisetty, M. Celniker, S. & Graveley, B.R. (2015) <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature14475.html">Genome-wide Identification of Zero Nucleotide Recursive Splicing in Drosophila</a>. Nature 521, May 21, 376-9.

<p>

Gerstein, M.B., et al (2014) <a href="http://www.nature.com/nature/journal/v512/n7515/full/nature13424.html">Comparative analysis of the transcriptome across distant species</a>. Nature 512, 445-448.

<p>

Chen, Z.X. et al (2014) <a href="http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=24985915">Comparative validation of the D. melanogaster modENCODE Transcriptome Annotation</a>. Genome Res. July;24(7):1209-23.

<p>

Brown, J.B. et al (many authors). 2014.<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12962.html"> Diversity and dynamics of the Drosophila transcriptome</a>. Nature, published online March 16, 2014.

<p>

Eipper-Mains JE et al (2013) <a href="http://www.ncbi.nlm.nih.gov/pubmed/23094851">Effects of cocaine and withdrawal on the mouse nucleus accumbens transcriptome</a>. Genes Brain Behav.12:21-33 

<p>

Smibert P, Miura P, Westholm JO, Shenker S, May G, Duff MO, Zhang D, Eads BD, Carlson J, Brown JB, Eisman R

C, Andrews J, Kaufman T, Cherbas P, Celniker SE, Graveley BR, Lai EC. (2012) <a href="http://www.cell.com/c

ell-reports/fulltext/S2211-1247(12)00021-6">Global patterns of tissue-specific alternative polyadenylation in Drosophila</a>. Cell Reports, Feb 23.

<p>

Yang L., Duff MO, Graveley BR, Carmichael GG and Chen L-L. 2011. <a href="http://genomebiology.com/2011/12/

2/R16">Genomewide characterization of long nonpolyadenylated RNAs</a>. Genome Biology, Feb 16. 

<p>

  Roy, S. et al (many authors). 2010. <a href="http://www.sciencemag.org/content/330/6012/1787.full">Identification of functional elements and regulatory circuits in Drosophila by large-scale data integration</a>. 

Science, Dec 24.

<p>
	
  Graveley,B.R, Brooks,A.N., Carlson,J.W., Duff,M.O., Landolin,J.M., Yang,L., et al (2010). <a href="https://www.nature.com/articles/nature09715">The developmental transcriptome of Drosophila melanogaster</a>. Nature, Dec 22. 

<p>

  Cherbas, L., et al (many authors). 2010. <a href="http://genome.cshlp.org/content/21/2/301.full">The transcriptional diversity of 25 Drosophila cell lines</a>. Genome Research, Dec 22. 

<p>

  Brooks, A.N., Yang. L., Duff, M.O., Hansen, K.D., Dudoit, S., Brenner, S.E., Graveley, B.R. 2010. <a href

="http://genome.cshlp.org/content/early/2010/10/04/gr.108662.110">Conservation of an RNA Regulatory Map between Drosophila and Mammals</a>. Genome Research, Oct 4.  

<p>

  McManus, C.J., Duff, M.O., Eipper-Mains, J., Graveley, B.R. 2010. <a href="http://www.pnas.org/content/107/29/12975.short">Global Analysis of Transplicing in Drosophila</a>. Proc Natl Acad Sci, July 1. 

<p>

  McManus, C.J., Coolon, J.D., Duff, M.O., Eipper-Mains, J., Graveley, B.R. and Wittkopp, P.J. 2010. <a href="http://www.pnas.org/content/107/29/12975.short">Regulatory Divergence in Drosophila Revealed by mRNA-Seq

</a>. Genome Research, 20(6):816-25. 

<p>

  Hale, C.R., Zhao, P., Olsen, S., Duff, M.O., Graveley, B.R., Wells, L., Terns, R.M., and Terns, M.P. 2009

. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19945378">RNA-guided RNA cleavage by a CRISPR RNA-Cas protein complex</a>. Cell, 139(5):945-56. Nov 25. 





<p>

<br>

<p>

<br>

<p>

<br>

 

 

<img src = "mouse-maze.jpg" width="250" height="136">

<img src = "mouse-in-maze.jpg" width="250" height="136"><br>

<img src = "bamdp.jpg" width="250" height="136">

<img src = "backup.jpg" width="250" height="136">



<h3>Mostly Reinforcement Learning Theory</h3>



<ul>



<li>Y Niv, MO Duff & P Dayan (2005) - <a href="http://www.behavioralandbrainfunctions.com/content/1/1/6">Dopamine, Uncertainty and TD Learning</a> - Behavioral and Brain Functions 1:6 (4 May 2005), doi:10.1186/1744-9081-1-6.<br><p>



<li>Duff, Chudova, Wold, Smyth, & Mjolsness. <a href="http://www.sigmoid.org/publications/icsb2005.pdf">Statistical inference of biologically-plausible dynamic regulatory networks with core-leaf topology</a>, ICSB, 2005.<br><p>



<li>Duff, M.O. <a href="http://www.aaai.org/Papers/ICML/2003/ICML03-020.pdf">Design for an optimal probe<a/>. Proceedings of the 20th International Conference on Machine Learning, 2003: 131-138.<br><p>



<li>Duff, M.O. <a href="http://www.aaai.org/Papers/ICML/2003/ICML03-021.pdf">Diffusion approximation for Bayesian Markov chains</a>. Proceedings of the 20th International Conference on Machine Learning, 2003: 139-146.<br><p>



<li>Duff, M.O. 

<a href="DuffThesis.pdf"> Optimal Learning: Computational procedures for Bayes-adaptive Markov decision processes</a>. Ph.D. Thesis, Dept. of Computer Science, Univ. of Massachusetts, Amherst, 2002.<br><p>

 

<li>Duff, M.O. <a href="duff_ai_stat_2001.pdf"> 

Monte-Carlo Algorithms for the Improvement of Finite-State Stochastic Controllers</a>.

 Eighth International Workshop on Artificial Intelligence and Statistics, Key West FL. <br><p>
	
	
<li>Duff, M.O. <a href="A_Control_Variable_Perspective_for_the_Optimal_Com.pdf">
	A Control Variable Perspective for the Optimal Combination of Truncated Corrected Returns</a>
	Research Note. Department of Computer Science, University of Massachusetts, Amherst.<br><p>
	

<li>Duff, M.O. & Barto, A. <a href="http://papers.nips.cc/paper/1230-local-bandit-approximation-for-optimal-learning-problems.pdf">Local bandit approximation for optimal learning problems</a>. Advances in Neural Information Processing Systems 9. 1997: 1019-1025.<br><p>



<li>Duff, M.O. <a href="Duff_1995_q_learning_bandits.pdf">Q-learning for bandit problems</a>. Proceedings of the 12th International Conference on Machine Learning, 1995: 209-217.<br><p>



<li>Bradke, S. & Duff, M.O. <a href="http://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=AFBA4907F489AA38EA2F9A3CAC069898?doi=10.1.1.32.9030&rep=rep1&type=pdf">Reinforcement learning methods for continuous-time Markov decision processes</a>. Advances in Neural Information Processing Systems 7. 1995: 393-400.<br><p>



<li>Duff, M.O. <a href="solving-bellmans-equation-by-solving-method-of-continuity.pdf">Solving Bellman's equation by the method of continuation</a>. Proceedings of the American Control Conference, 1994.: 2671-2682.<br><p>



<li>Barto A. & Duff, M.O. <a href="NIPS-1993-monte-carlo-matrix-inversion-and-reinforcement-learning-Paper.pdf">Monte-Carlo matrix inversion and reinforcement learning</a>. Advances in Neural Information Processing Systems, vol 6 1994: 687-694.<br><p>



<li>Duff, M.O. Backpropagation and Bach's 5th cello suite (Sarabande). Proceedings of the International Joint Conference on Neural Networks.<br><p>



<li>Szilagyi, M.O., Duff, M., & Yakowitz, S. Procedure for electron and ion lens optimization. Applied Physics Letters.<br><p>



</ul>



<p>

<br>



<!--

<a href="chrono_bio.html"><font color="#3399FF">My Erdos number is 3.</font></a>





<a href="older_papers.html"><font color="#3399FF">Other things (Machine Learning and Reinforcement Learning Theory)</font></a>



 

 

 

-->

 

<p>

<br>



<a href="chrono_bio.html"><font color="#3399FF">My Erdos number is 3.</font></a>





<br>



</font>



<TD width="20%">

</TABLE>

</body>





  </html>

	
